shared-helpers/src/types/backend-swagger.ts

```diff
@@ -6962,12 +7049,6 @@ export enum EnumUnitGroupAmiLevelMonthlyRentDeterminationType {
   "flatRent" = "flatRent",
   "percentageOfIncome" = "percentageOfIncome",
 }
-export enum HomeTypeEnum {
```

Comment:
question: Why is this getting deleted?

Reply:
I think it was duplicated, that enum is still in the swagger file, just that set of lines is gone